sat/engine/runtime/api/api.go

```diff
@@ -48,15 +48,10 @@ func (d *defaultAPI) HostFunctions() []runtime.HostFn {
 		d.AddFFIVariableHandler(),
 		d.FetchURLHandler(),
 		d.GraphQLQueryHandler(),
```

Comment:
Is graphql sticking around for any particular reason or did we just miss it?

Reply:
I would like GraphQL removed—it's another thing we have to maintain and can be accomplished via HTTP.

Reply:
I was going to leave it since it literally _is_ the HTTP capability under the hood, but I don't have strong feelings.

Main reason is that tons of companies expose their APIs as GraphQL and so it's a nicety to support it directly.